docs/user-guide/walkthrough/k8s201.md

```diff
@@ -1,122 +1,153 @@
----
----
+---
+---
 
-## Labels, Replication Controllers, Services and Health Checking
+## Labels, Deployments, Services and Health Checking
 
 If you went through [Kubernetes 101](/docs/user-guide/walkthrough/), you learned about kubectl, pods, volumes, and multiple containers.
 For Kubernetes 201, we will pick up where 101 left off and cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
 scaling.
 
 In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/kubernetes/kubernetes/releases) or [the source](https://github.com/kubernetes/kubernetes).
 
-* TOC
+* TOC
 {:toc}
 
-
+
 ## Labels
 
 Having already learned about Pods and how to create them, you may be struck by an urge to create many, many pods.  Please do!  But eventually you will need a system to organize these pods into groups.  The system for achieving this in Kubernetes is Labels.  Labels are key-value pairs that are attached to each object in Kubernetes.  Label selectors can be passed along with a RESTful `list` request to the apiserver to retrieve a list of objects which match that label selector.
 
 To add a label, add a labels section under metadata in the pod definition:
 
-```yaml
+```yaml
   labels:
     app: nginx
-```
-
+```
+
 For example, here is the nginx pod definition with labels ([pod-nginx-with-label.yaml](/docs/user-guide/walkthrough/pod-nginx-with-label.yaml)):
-
+
 {% include code.html language="yaml" file="pod-nginx-with-label.yaml" ghlink="/docs/user-guide/walkthrough/pod-nginx-with-label.yaml" %}
 
 Create the labeled pod ([pod-nginx-with-label.yaml](/docs/user-guide/walkthrough/pod-nginx-with-label.yaml)):
 
-```shell
-$ kubectl create -f docs/user-guide/walkthrough/pod-nginx-with-label.yaml
-```
-
+```shell
+kubectl create -f docs/user-guide/walkthrough/pod-nginx-with-label.yaml
+```
+
 List all pods with the label `app=nginx`:
 
-```shell
-$ kubectl get pods -l app=nginx
-```
-
-For more information, see [Labels](/docs/user-guide/labels).
-They are a core concept used by two additional Kubernetes building blocks: Replication Controllers and Services.
+```shell
+kubectl get pods -l app=nginx
+```
+
+For more information, see [Labels](/docs/user-guide/labels/).
+They are a core concept used by two additional Kubernetes building blocks: Deployments and Services.
+
+
+## Deployments
+
+OK, now you know how to make awesome, multi-container, labeled pods and you want to use them to build an application, you might be tempted to just start building a whole bunch of individual pods, but if you do that, a whole host of operational concerns pop up.  For example: how will you scale the number of pods up or down?  how will you rollout a new release?

```

Comment:
s/how will you roll out/How will you roll out
s/OK, now you know/Now that you know